client/signup/steps/user/index.jsx

```diff
@@ -40,56 +54,58 @@ export default React.createClass( {
 			1 === signupUtils.getFlowSteps( props.flowName ).length &&
 			'userfirst' !== props.flowName
 		) {
-			subHeaderText = this.translate( 'Welcome to the wonderful WordPress.com community' );
+			subHeaderText = this.props.translate( 'Welcome to the wonderful WordPress.com community' );
 		}
 
 		this.setState( { subHeaderText: subHeaderText } );
-	},
+	}
 
-	save( form ) {
+	save = ( form ) => {
 		SignupActions.saveSignupStep( {
 			stepName: this.props.stepName,
 			form: form
 		} );
-	},
+	};
 
-	submitForm( form, userData, analyticsData ) {
-		let flowName = this.props.flowName,
-			queryArgs = {};
+	submitForm = ( form, userData, analyticsData ) => {
+		const queryArgs = {};
```

Comment:
Not absolutely the point of the PR, but if `queryArgs` is only used to contain `jetpackRedirect`, I'd avoid the multi-statement process and so something like

```js
const queryArgs = {
  jetpackRedirect: get( this.props, 'queryObject.jetpack_redirect' )
}; // assuming lodash#get, which will return undefined if prop not found, which is what we want
```